people/yumao/page.en.md

```diff
@@ -0,0 +1,152 @@
+---
+name: Yumao
+info:
+    alias: Yumao, Zhangqianye, Yunyu
+    location: Shanghai
+---
+
+> "I will transform myself into gentle breeze and rain, then surround you all along."
+>
+> By Yumao
+>
+> "It feels like that you are hiding in crystal alone, all I can see is your shattered figure. "
+>
+> By Neko
+
+## Description
+
+Yumao (Zhangqianye) is a trans girl, who is gentle, smart, tough and lovely.
+
+She introduced herself in her profile:
+Perhaps Yumao doesn't like taking part in group activities.
+I love white and red, as well as that kind of rainbow in Glitch.
+And I have a sweet tooth, I will feel happy for a long time while holding a packet of white rabbit creamy candies.
+I love cats, and adopt a cat called Ayaka.
+
+She was an admiral of *Kancolle* , and her first action figure was Destroyer Akizuki in *kancolle*.
+
+She loved time with silent and ease.
+For example, she put on the headphones and drank hot coffee in café, then watched out of the window and observed people came and left. This would make her relaxed.
+Sometimes, she had some special thoughts.
+She made wishes to meteors occasionally, she said: "It's not because I believe that, but in that it's free. And there's no evidence shows it doesn't work."
```

Comment:
She made wishes to meteors occasionally. She said: "It's not because I believe that, but in that it's free. And there's no evidence showing it doesn't work."